test/unit/controllers/notifications/NotificationCallbackControllerSpec.scala

```diff
@@ -25,16 +25,16 @@ import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.mockito.MockitoSugar
 import play.api.test.Helpers._
 import services.NotificationService
-import base.ControllerUnitSpec
+import base.{ControllerUnitSpec, SfusMetricsMock}
 
 import scala.concurrent.ExecutionContext.global
 import scala.concurrent.Future
 import scala.xml.NodeSeq
 
-class NotificationCallbackControllerSpec extends ControllerUnitSpec with MockitoSugar with ScalaFutures with BeforeAndAfterEach {
+class NotificationCallbackControllerSpec extends ControllerUnitSpec with MockitoSugar with ScalaFutures with BeforeAndAfterEach with SfusMetricsMock {
```

Comment:
Do we need to check that the mocked metrics are called?  E.g. timer is stopped for both success and error paths; counter only incremented on success, etc.

Reply:
counter is incremented only on success, because we shouldn't increment it for failures.

About timer I though that it's good to stop this, but you're right, we're not stopping this for failures in our services.

About tests that's good point. We didn't tested this for exports service, but it's worth to add those checks.

Thanks

Reply:
Added checks and missing test case :)